doc/neps/nep-0028-website-redesign.rst

```diff
@@ -0,0 +1,302 @@
+have a similar deployment to GitHub Pages or Netlify.
+
+Analytics
+~~~~~~~~~
+
+It's benefical to maintainers to know how many visitors are coming to
+numpy.org. Google Analytics offers visitor counts and locations. This will
+help to support and deploy more strategically, and help maintainers
+understand where traffic is coming from.
+
+Google Analytics is free. A script, provided by Google, must be added to the home page.
+
+Website Structure
+~~~~~~~~~~~~~~~~~
+
+We aim to keep the first version of the new website small in terms of amount
+of content. New pages can be added later on, it's more important right now to
+get the site design right and get some essential information up. Note that in
+the second half of 2019 we expect to get 1 or 2 tech writers involved in the
+project via Google Season of Docs. They will likely help improve the content
+and organization of that content.
+
+We propose the following structure:
+
+0. Front page: essentials of what NumPy is (compare e.g. jupyter.org), one or
+   a couple key user stories (compare e.g. julialang.org)
+1. Install
+2. About Us
+3. Community
+4. Documentation
+5. Array computing
+6. Support/Donate
+
+There may be a few other pages, e.g. a page on performance, that are linked
+from one of the main pages.
+
+Stakeholder Content
+~~~~~~~~~~~~~~~~~~~
+
+This should have as little content as possible *within the site*. Somewhere
+on the site we should link out to content that's specific to:
+
+- beginning users (quickstart, tutorial)
+- advanced users
+- educators
+- packagers
+- package authors that depend on NumPy
+- funders (governance, roadmap)
+
+Translation (multilingual / i18n)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+NumPy has users all over the world. Most of those users are not native
+English speakers, and many don't speak English well or at all. Therefore
+having content in multiple languages is potentially addressing a large unmet
+need. It would likely also help make the NumPy project more diverse and
+welcoming.
+
+On the other hand, there are good reasons why few projects have a
+multi-lingual site. It's potentially a lot of extra work. Extra work for
+maintainers is costly - they're already struggling to keep up with the work
+load. Therefore we have to very carefully consider whether a multi-lingual
+site is feasible and weight costs and benefits.
+
+We start with an assertion: maintaining translations of all documentation, or
+even the whole user guide, as part of the NumPy project is not feasible. One
+simply has to look at the volume of our documentation and the frequency with
+which we change it to realize that that's the case. Perhaps it will be
+feasible though to translate just the top-level pages of the website. Those
+do not change very often, and it will be a limited amount of content (order
+of magnitude 5-10 pages of text).
+
+We propose the following requirements for adding a language:
+
+- The language must have a dedicated maintainer
```

Comment:
would be better to have multiple, the bus factor effect in this regime could be ever more present

Reply:
(Coming from a very small language, it was always frustrating to see that after the initial inertia and team disappeared the translated pages in that language are both outdated and incomplete, and can't remember any examples that relied on people doing the translation rather than an automated framework. )

Reply:
Agreed, more is better. Let's see what others think - should we require at least two people?

Reply:
I would think a clear policy when to disable a translation would be preferable. Even if there are two volunteers right now, there is no guarantee that 1-5-10 years down the line still there is. And removing things is always difficult, having a clear guideline would be then useful (e.g. remove by default once there isn't two people around to actively maintain it).

Reply:
> I would think a clear policy when to disable a translation would be preferable.

agreed. I wrote a bit about that, but I should probably make it more explicit.

Reply:
I added a section with a policy. thanks @bsipocz